### PR TITLE
Fixed crash when gene hasnt a primary_transcript

### PR DESCRIPTION
**Pull request by mhkc**
This fixes an issue when the variant view would crash if a gene hasn't a primary transcript

**Review:**
- [x] code approved by CR
- [x] tests executed by MJ

### CODE REVIEW

File: scout/server/blueprints/variant/templates/variant/tx_overview.html [hunk 1]
```diff
@@ -69,7 +69,7 @@
                   <td class="d-flex align-items-center">
                     <div> <!-- ID col-->
                       <span class="text"> {{ transcript.transcript_id }}</span>
-                      {% if gene.common and transcript.refseq_id in gene.common.primary_transcripts %}
+                      {% if (primary_transcripts in gene.common) and (gene.common and transcript.refseq_id in gene.common.primary_transcripts) %}
```
Comment (northwestwitch) [inline scout/server/blueprints/variant/templates/variant/tx_overview.html:72]:
```suggestion
                      {% if gene.common and 'primary_transcripts' in gene.common and transcript.refseq_id in gene.common.primary_transcripts %}
```

Comment (northwestwitch) [reply scout/server/blueprints/variant/templates/variant/tx_overview.html:72]: Jinja is slight different from python and to check that a key exists in a dictionary you check it either with

`if 'key' in dict ` or with `if dict.key`

We've found this out with last week, remember @dnil ? 🤣

**Review by northwestwitch — COMMENTED** [review]
I have a suggestion but haven't tested it. Could test it later!

File: scout/server/blueprints/variant/templates/variant/tx_overview.html (hunk 1 above)
Comment (dnil) [reply scout/server/blueprints/variant/templates/variant/tx_overview.html:72]: 😅 Yes, that was not totally obvious. Flask2 is like the Dragos, tough on the tough?

Comment (dnil) [reply scout/server/blueprints/variant/templates/variant/tx_overview.html:72]: https://www.naspbokhandel.se/pub_images/large/9789176217115_1.Jpeg?timestamp=1616910123

Comment (mhkc) [reply scout/server/blueprints/variant/templates/variant/tx_overview.html:72]: @northwestwitch Ill try that solution.

**Comment by mhkc** [comment]
@northwestwitch your solution works. I have committed it.

**Review by northwestwitch — APPROVED** [review]
👍🏻

**Pull request merged**